bucket/geforce-now.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "nightly",
+    "description": "NVIDIA’s cloud-based game streaming service",
+    "homepage": "https://www.nvidia.com/en-us/geforce-now/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.nvidia.com/en-us/geforce-now/terms-of-use/"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.nvidia.com/gfnpc/GeForceNOW-release.exe#/dl.7z"
+        }
+    },
+    "extract_dir": "GeforceNOW",
+    "bin": "CEF\\GeForceNOW.exe",
+    "shortcuts": [
+        [
+            "CEF\\GeForceNOW.exe",
+            "GeForce NOW"
+        ]
+    ]
+}
```

Comment:
Can you add autoupdate support?

Reply:
I cannot find a trackable update log. Will try to add `autoupdate` via *UpdateTracker*.